corehq/apps/app_manager/views.py

```diff
@@ -302,16 +302,6 @@ def xform_display(request, domain, form_unique_id):
         return json_response(questions)
 
 
-@require_can_edit_apps
-def form_casexml(request, domain, form_unique_id):
-    try:
-        form, app = Form.get_form(form_unique_id, and_app=True)
-    except ResourceNotFound:
-        raise Http404()
-    if domain != app.domain:
-        raise Http404()
-    return HttpResponse(form.create_casexml())
```

Comment:
This was just a legacy view. There are not `create_casexml` functions anywhere in the codebase.